lib/routes/account.js

```diff
@@ -487,13 +489,20 @@ module.exports = function (
           if (shouldSendNewDeviceLoginEmail) {
             // The response doesn't have to wait for this,
```

Comment:
The response doesn't depend on returning the promise of the getGeoData response, but the tests do. This comment is no longer valid.